modules/virt-downloading-virtctl-client.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// virt/install/virt-enabling-virtctl.adoc
+
+[id="virt-downloading-virtctl-client_{context}"]
+= Downloading the virtctl client
+
+Download the `virtctl` client provided in a pod in the cluster.
```

Comment:
I feel `in a pod in the cluster` this is misleading, since subsequent procedure is not talking about any pods.

Reply:
virtctl binaries are served from a pod in the cluster. the statement refers that pod. do you think it is misleading?

Reply:
@erkanerol I understand that. What I meant is, this statement `Download the virtctl client provided in a pod in the cluster` together with the procedure (that does not refer to pods or pod related commands) is confusing. I hope I am able to convey my view.

Reply:
We can improve it like `Download the virtctl client from a route of the cluster`

Reply:
@dbasunag @erkanerol What about this rewrite?

`Download the virtctl client using the link provided in the ConsoleCLIDownload custom resource (CR).`

Reply:
Sounds good to me. Thanks @sabrinajess and @erkanerol .